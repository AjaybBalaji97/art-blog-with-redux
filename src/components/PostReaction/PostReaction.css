.clap{
    width:30px;
    height:30px;
}
.heart{
    float:right;
    width:30px;
    height:30px;
}
p{
    display:inline;
    size:18px;
}