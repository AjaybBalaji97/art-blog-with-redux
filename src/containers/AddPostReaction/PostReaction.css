
.clap{
    width:30px;
    height:30px;
    margin:10px
}
p{
    display: inline;
    vertical-align: super;
    font-size:15px;
}
.heart{
    float:right;
    width:30px;
    height:30px;
    margin:10px
}
