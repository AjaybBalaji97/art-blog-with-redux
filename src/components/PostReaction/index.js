import React, { Component } from 'react';
//import BlackHeartImage from './Icons/heart-black.svg'
//import RedHeartImage from './Icons/heart-red.svg';
//import ClappingImage from './Icons/clapping.svg';
import './PostReaction.css';
import { throws } from 'assert';
class PostReaction extends Component {
    render() {
      const liked = this.props.liked?`./Icons/heart-red.svg`:`./Icons/heart-black.svg`;

  
      return (
        <div className="main" >
            <span><img alt="clap" src={`./Icons/clapping.svg`} className='clap' onClick={this.props.handleClapClick}/>
            <p>{this.props.claps}</p>
            {
              <img alt="heart" src={liked} className='heart' onClick={this.props.handleLikeClick} />
            }
            </span>
        </div>
      );
    }
  }
  export default PostReaction;