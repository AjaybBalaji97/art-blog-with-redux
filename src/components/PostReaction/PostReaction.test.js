import React from 'react';
import renderer from 'react-test-renderer';
import PostReaction from '.';
import {shallow} from 'enzyme';


describe('PostReaction',()=>{
    it('should render header',()=>{
        const tree=renderer.create(<PostReaction  />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});