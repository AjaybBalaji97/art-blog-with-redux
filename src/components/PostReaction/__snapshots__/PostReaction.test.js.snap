// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostReaction should render header 1`] = `
<div
  className="main"
>
  <img
    alt="clap"
    className="clap"
    src="./Icons/clapping.svg"
  />
  <p />
  <img
    alt="heart"
    className="heart"
    src="./Icons/heart-black.svg"
  />
</div>
`;
